docs/community/events.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Notable Events
+title: Past Events
```

Comment:
On this page, there are double `past events` headers. Can you remove the one below this? 
![image](https://user-images.githubusercontent.com/44212783/153053932-321fc2da-ac21-48ec-96d5-72985cde4ac3.png)

Not a comment for you, but this page has also not been updated in months.